scripts/maven-generate.sh

```diff
@@ -10,6 +10,8 @@ if [[ -f ${MAVEN_SETTINGS_FILE} ]]; then
     exit 0
 fi
 
+mkdir "${MAVEN_GENERATE_DIRECTORY}"
```

Comment:
can you specify what issue you were facing 
and how moving this line helped ?

Reply:
so basically when the user provides a settings.xml file in the workspace that time, it goes to the if block at line no. 16 which then would do a cp command on a non-existing directory & fail, to fix that, we need to do mkdir anyways when doing cat later on if the workspace isnt provided since its provided now, we can just do it like so instead.

Reply:
Got it